### PR TITLE
Add an arrow button in the infoview that reverses the order of the lists under Tactic State

### PR DESCRIPTION
**Pull request by lovettchris**
This is part one of resurrecting tactic state filtering, this is an easy one that can be done on the client side without any LSP changes and is related to issue https://github.com/leanprover/vscode-lean4/issues/76.  See demo video:

https://user-images.githubusercontent.com/18707114/168698204-65aa24e0-f971-4d60-ae7e-4522cd70edb3.mp4

### CODE REVIEW

File: lean4-infoview/src/infoview/info.tsx [hunk 1]
```diff
@@ -148,7 +152,7 @@ export function InfoDisplay(props0: InfoDisplayProps) {
                         Expected type
```
Comment (gebner) [inline lean4-infoview/src/infoview/info.tsx:152]: The arrow should be here as well

File: lean4-infoview/src/infoview/goals.tsx [hunk 2]
```diff
@@ -29,25 +29,31 @@ export function goalsToString(goals: InteractiveGoals): string {
...
     const prefix = goal.goalPrefix ?? '⊢ '
+    const hyps = reverse  ? goal.hyps.slice().reverse() : goal.hyps;
     return <div className="font-code tl pre-wrap">
         <ul className="list pl0">
             {goal.userName && <li key={'case'}><strong className="goal-case">case </strong>{goal.userName}</li>}
-            {goal.hyps.map ((h, i) => {
+            {reverse &&
+            <li key={'goal'}>
+                <strong className="goal-vdash">{prefix}</strong><InteractiveCode pos={pos} fmt={goal.type} />
+            </li>}
```
[the remark this answers is not in the document]
Comment (lovettchris) [reply lean4-infoview/src/infoview/goals.tsx]: Done

File: lean4-infoview/src/infoview/goals.tsx [hunk 3]
```diff
@@ -56,8 +62,9 @@ export function Goals({pos, goals}: {pos: DocumentPosition, goals: InteractiveGo
     if (goals.goals.length === 0) {
         return <>Goals accomplished 🎉</>
     } else {
+        const sorted = reverseOrder ? goals.goals.reverse() : goals.goals;
```
[the remark this answers is not in the document]
Comment (lovettchris) [reply lean4-infoview/src/infoview/goals.tsx]: ok, fixed.

File: lean4-infoview/src/infoview/info.tsx (hunk 1 above)
Comment (lovettchris) [reply lean4-infoview/src/infoview/info.tsx]: right, fixed.

**Pull request merged**